osmosis-1/osmosis-frontier.assetlist.json

```diff
@@ -1395,6 +1395,34 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/juno/images/rac.svg"
       }
     },
+    {
+      "description": "Building Consensus among Hong Kong People",
+      "type_asset": "cw20",
+      "address": "juno1tdjwrqmnztn2j3sj2ln9xnyps5hs48q3ddwjrz7jpv6mskappjys5czd49",
+      "denom_units": [
+        {
+          "denom": "ibc/52E12CF5CA2BB903D84F5298B4BFD725D66CAB95E09AA4FC75B2904CA5485FEB",
+          "exponent": 0
+        },
+        {
+          "denom": "dhk",
+          "exponent": 0
```

Comment:
is this denom meant to be 0? if so, 'dhk' should just be an alias under the first denom units.

Reply:
updated